i18n/es.jsonp.js

```diff
@@ -178,19 +179,19 @@ document.localeJson = {
       "dialog-serviceworker-unsupported-title": "Modo ServiceWorker no disponible",
-      "dialog-serviceworker-unsupported-message": "<p>Desafortunadamente, su navegador no parece soportar el modo ServiceWorker, que ahora es el modo predeterminado para esta aplicación.</p><p>Puede continuar usando la aplicación en el modo JQuery (ahora obsoleto), pero tenga en cuenta que este modo sólo funciona bien con los archivos ZIM que tienen contenido estático, como los archivos ZIM de Wikipedia / Wikimedia o Stackexchange.</p><p>Si puede, le recomendamos que actualice su navegador a una versión que soporte el modo ServiceWorker.</p>",
-      "dialog-serviceworker-unsupported-fallback": "Use modo JQuery",
+      "dialog-serviceworker-unsupported-message": "<p>Desafortunadamente, su navegador no parece soportar el modo ServiceWorker, que ahora es el modo predeterminado para esta aplicación.</p><p>Puede continuar usando la aplicación en el modo Seguro, pero tenga en cuenta que este modo sólo funciona bien con los archivos ZIM que tienen contenido estático, como los archivos ZIM de Wikipedia / Wikimedia o Stackexchange.</p><p>Si puede, le recomendamos que actualice su navegador a una versión que soporte el modo ServiceWorker.</p>",
+      "dialog-serviceworker-unsupported-fallback": "Use modo Seguro",
       "dialog-serviceworker-defaultmodechange-title": "Cambio del modo de inyección de contenido predeterminado",
-      "dialog-serviceworker-defaultmodechange-message": "<p>Le hemos cambiado al modo ServiceWorker (ahora es el modo predeterminado). Soporta más tipos de archivo ZIM y es mucho más robusto.</p><p>Si tiene problemas con este modo, puede volver al modo JQuery (ahora obsoleto). En tal caso, por favor, informe de los problemas que ha experimentado (véase la página Información).</p>",
+      "dialog-serviceworker-defaultmodechange-message": "<p>Le hemos cambiado al modo ServiceWorker (ahora es el modo predeterminado). Soporta más tipos de archivo ZIM y es mucho más robusto.</p><p>Si tiene problemas con este modo, puede volver al modo Seguro . En tal caso, por favor, informe de los problemas que ha experimentado (véase la página Información).</p>",
       "dialog-slideawaycheck-message": "Esta configuración puede no funcionar correctamente en Firefox OS. Si encuentra que algunos enlaces ZIM no responden, se recomienda desactivar esta configuración.",
-      "dialog-sourceverification-alert": "¿Proviene este archivo ZIM de una fuente de confianza?\nDe no ser así, aún puede leer el archivo en Modo Seguro (también conocido como modo JQuery). Al cerrar esta ventana también se abre el archivo en Modo Seguro. Esta verificación puede desactivarse en Opciones avanzadas.",
+      "dialog-sourceverification-alert": "¿Proviene este archivo ZIM de una fuente de confianza?\nDe no ser así, aún puede leer el archivo en Modo Seguro. Al cerrar esta ventana también se abre el archivo en Modo Seguro. Esta verificación puede desactivarse en Opciones avanzadas.",
```

Comment:
Elsewhere the capitalization is "modo Seguro", but here it is "Modo Seguro". I think change this to "modo Seguro" for consistency.